telegram_packs/other.txt

```diff
@@ -0,0 +1,88 @@
+other/biFrog|🏳️
```

Comment:
:transgender_flag: ? closest in terms of thematic i think

Reply:
yeah
I already changed that flag to the pride one in another repo, but not here.
Trans seems more fitting though so will use that. :+1: